src/main/cpp/pop/Household.h

```diff
@@ -0,0 +1,24 @@
+#ifndef POPULATION_HOUSEHOLD_H_INCLUDED
+#define POPULATION_HOUSEHOLD_H_INCLUDED
+
+/**
+ * @file
+ * Header file for reference households
+ */
+
+#include <vector>
+#include <boost/property_tree/ptree.hpp>
+
+namespace stride {
+namespace population {
+
+// A list of ages.
+using Household = std::vector<int>;
```

Comment:
You say that a `Household` is a list of ages in the comment. I don't think that's entirely obvious. Maybe should find a better name for `Household`.